src/consensus/aft/raft.h

```diff
@@ -63,6 +63,8 @@
 #define RAFT_TRACE_JSON_OUT(json_object) \
   CCF_LOG_OUT(DEBUG, "raft_trace") << json_object
 
+#define LOG_ROLLBACK_FMT CCF_LOG_FMT(INFO, "rollback")
```

Comment:
```suggestion
#define LOG_ROLLBACK_INFO_FMT CCF_LOG_FMT(INFO, "rollback")
```

I think having the log level consistently in the macro is really helpful when glancing at the code to find out what we expect to see logged where.